asab/storage/upsertor.py

```diff
@@ -144,21 +144,24 @@ async def execute(self, custom_data: typing.Optional[dict] = None):
 
 
 	async def _webhook(self, data: dict):
-		assert self.Storage.WebhookURI is not None
+		assert self.Storage.WebhookURIs is not None
```

Comment:
This should be rather a proactor routine *per* URI; it will be much more defensive against issues with receivers.

Reply:
@ateska I refactored it to use proactor, although I'm not sure if this is what you meant. Can you check it now?